test/rules/require-capitalized-constructors.js

```diff
@@ -4,25 +4,49 @@ var assert = require('assert');
 describe('rules/require-capitalized-constructors', function() {
     var checker;
 
-    beforeEach(function() {
-        checker = new Checker();
-        checker.registerDefaultRules();
-        checker.configure({ requireCapitalizedConstructors: true });
-    });
+    function baseCases() {
+        it('should report uncapitalized construction', function() {
+            assert(checker.checkString('var x = new y();').getErrorCount() === 1);
+        });
 
-    it('should report uncapitalized construction', function() {
-        assert(checker.checkString('var x = new y();').getErrorCount() === 1);
-    });
+        it('should not report capitalized construction', function() {
+            assert(checker.checkString('var x = new Y();').isEmpty());
+        });
 
-    it('should not report capitalized construction', function() {
-        assert(checker.checkString('var x = new Y();').isEmpty());
-    });
+        it('should not report member expression construction', function() {
+            assert(checker.checkString('var x = new ns.y();').isEmpty());
+        });
+
+        it('should not report construction with "this" keyword', function() {
+            assert(checker.checkString('var x = new this();').isEmpty());
+        });
+    }
 
-    it('should not report member expression construction', function() {
-        assert(checker.checkString('var x = new ns.y();').isEmpty());
+    describe('with `true` value', function() {
+        beforeEach(function() {
+            checker = new Checker();
+            checker.registerDefaultRules();
+            checker.configure({ requireCapitalizedConstructors: true });
+        });
+
+        baseCases();
     });
 
-    it('should not report construction with "this" keyword', function() {
-        assert(checker.checkString('var x = new this();').isEmpty());
+    describe('with `allExcept` value', function() {
+        beforeEach(function() {
+            checker = new Checker();
```

Comment:
move this and the next line back into a top level `beforeEach`, and only have the `configure` call in each inner `beforeEach`.